### DISCUSSION TITLE
Allow users to collapse sidebar

### DISCUSSION THREAD

**Discussion by hirehamir** [bcakmakoglu/vue-flow#1723] [Ideas]
The sidebar uses up a lot of room in portrait mode. It'd be nice if users could collapse it, ideally with a customizable shortcut.

**Comment by hirehamir** [comment]
Ctrl + B or
Ctrl + Shift + S or
Ctrl + Shift + E seem like sane default values for a collapse sidebar shortcut.

**Comment by hirehamir** [comment]
![image](https://github.com/user-attachments/assets/97d8e1ea-fca3-42fa-b24a-6992d44cbc84)

**Reply by bcakmakoglu** [reply]
You can change the size using the cursor and just slide it around.
This won't be "fixed" since it's a non-issue.

**Reply by hirehamir** [reply]
I think you're referring to the code sidebar.

I'm referring to the page's lefthand sidebar, which has the table of contents. My cursor doesn't change when I try to resize it.

**Reply by bcakmakoglu** [reply]
I see, still not something that'll be "fixed" since it's just the VitePress layout and not some custom implementation of my own and I don't see any issue with it either.
